### DISCUSSION TITLE
How to blend Projected Material with current mesh material?

### DISCUSSION THREAD

**Discussion by sapoluri** [marcofugaro/three-projected-material#48] [Q&A]
Am I missing an obvious way to do this in ThreeJS?

Do I really have to write a custom shader to do this?

I tried using THREE.SceneUtils.createMultiMaterialObject and project on only one of the objects in the group for which projected material is the material. Tried ShaderMaterial. Tried passing multi materials to the mesh with one being projected material and that doesn't work.

How do I achieve this?

**Comment by marcofugaro** [comment]
Since version `2.2.0` you can now use the ProjectedMaterial with common material using groups. Check out the [multiple projections example](https://marcofugaro.github.io/three-projected-material/multiple-projections)
